auditlog/tests/common.py

```diff
@@ -0,0 +1,28 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class AuditLogRuleCommon(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.models = set()
+
+    @classmethod
+    def create_rule(cls, vals):
+        rule = cls.env["auditlog.rule"].with_context(tracking_disable=True).create(vals)
+        # Keep track of patched models
+        cls.models |= set(rule.model_id.mapped("model"))
+        return rule
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.env["auditlog.rule"].search([]).unsubscribe()
```

Comment:
```
```suggestion
        for rule in cls.env["auditlog.rule"].search([]):
            try:
                rule.unsubscribe()
            except KeyError:
                # Model not loaded yet
                continue
```

Reply:
Could you give me an example of how to test this behavior? How can a model not be loaded?

Reply:
Oh, if you run tests on a database on which rules have been manually configured, or on which a module was installed earlier that includes data for rules for models that are not loaded when these tests load.

Reply:
I don't think that's a valid test environment.

Reply:
Since `auditlog` does not depend on `account`, it is impossible for a clean test environment to have an `auditlog.rule` on an `account.move` record on first run. Though it would make development a lot easier if tests could be repeatedly run without dropping and recreating the database every time, it often simply doesn't work. Module installations can override methods or change field permissions in such a way that it becomes impossible for subsequent test run to succeed.

That said, forward-compatibility is a laudable goal. My main hesitation in adding this sort of compatibility code is that it is by definition unable to be tested; permanently red in code coverage.

Reply:
That seems a small price to pay for the convenience of using the same database for both automated and manual testing.

Reply:
TIL `pragma: no cover` which sufficiently addresses my concern; added `KeyError` guard to `tearDownClass`.